features/Context/FixturesContext.php

```diff
@@ -971,12 +971,8 @@ public function theOptionsOfProductsShouldBe($attribute, $products, TableNode $t
         $this->getMainContext()->getSubcontext('hook')->clearUOW();
         foreach ($this->listToArray($products) as $identifier) {
             $productValue = $this->getProductValue($identifier, strtolower($attribute));
-            $options      = $productValue->getOptions();
-            $optionCodes  = $options->map(
-                function ($option) {
-                    return $option->getCode();
-                }
-            );
+            $options      = $productValue->getData();
+            $optionCodes  = array_map(function($option) { return $option->getCode(); },  $options);
```

Comment:
I know it is purely personal code style, but I really don't like this one line writing (and it is not PSR-2 compliant).